persistence/src/main/java/fi/otavanopisto/pyramus/dao/base/DefaultsDAO.java

```diff
@@ -26,6 +26,12 @@ public Defaults updateDefaultBaseTimeUnit(EducationalTimeUnit defaultEducational
     EntityManager entityManager = getEntityManager();
 
     Defaults defaults = getDefaults();
+    
+    if(defaults == null) {
```

Comment:
Defaults can be null in the other two methods of this class, too. Might as well fix all three places while at it.